### PR TITLE
Added interface to get/set the art on paintings, and other paintings stuff

### PR DESCRIPTION
**Pull request by CelticMinstrel**
Added interface to get/set the art and facing direction on paintings, and expanded painting break events to catch more cases (including fire and lightning); removed PaintingBreakByWorldEvent since it's identical to its superclass

### CODE REVIEW

**Comment by CelticMinstrel** [comment]
[Associated CraftBukkit pull request](https://github.com/Bukkit/CraftBukkit/pull/449)

File: src/main/java/org/bukkit/Art.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    public int getBlockHeight() {
+        return height;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static Art getById(int id) {
+        return ids.get(ids);
```
Comment (SolraBizna) [inline src/main/java/org/bukkit/Art.java]: This should be ids.get(id); or else getById always returns null.

File: src/main/java/org/bukkit/Art.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
+package org.bukkit;
+
+import java.util.HashMap;
+
+/**
+ * Represents the art on a painting
+ */
+public enum Art {
+    KEBAB(0,1,1),
```
Comment (SolraBizna) [inline src/main/java/org/bukkit/Art.java:9]: These should be CamelCase rather than ALLCAPS in order for the trick in the CraftBukkit counterpart to work.

Comment (CelticMinstrel) [inline src/main/java/org/bukkit/Art.java:9]: I don't think so; the Minecraft counterpart is an enum with the exact same constant names; yes, each constant stores a camel case version of itself, but the constant name itself is uppercase.

**Comment by CelticMinstrel** [comment]
Thanks for noticing the typo; fixed!

File: src/main/java/org/bukkit/Art.java (hunk 2 above)
Comment (SolraBizna) [inline src/main/java/org/bukkit/Art.java:9]: I remember having to change the mnemonics before getArt would return anything but null. I could be wrong... if I were awake, I'd test it. (Donate to your local chapter of Coders Against Sleep Posting today!)

Comment (CelticMinstrel) [inline src/main/java/org/bukkit/Art.java:9]: Maybe I'll donate to the CASPers later, thanks. :P

**Comment by EvilSeph** [comment]
Applied manually, thanks!

**Pull request closed without merging**